### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by traylenator**
First release of puppet-mlocate

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-mlocate",
-  "version": "0.0.0-rc1",
+  "version": "0.0.1",
```
Comment (Dan33l) [inline metadata.json]: If this code will be released on the forge, following semver should produce a version `1.0.0`:
https://semver.org/
> Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.

**Comment by traylenator** [comment]
All change assuming the branch name is irrelevant which I think it is.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [v0.0.1](https://github.com/voxpupuli/puppet-mlocate/tree/v0.0.1) (2019-11-27)
```
Comment (Dan33l) [inline CHANGELOG.md]: Please regenerate the `CHANGELOG.md` file.

**Comment by traylenator** [comment]
@Dan33l thanks for review.

Release published : https://forge.puppet.com/puppet/mlocate/1.0.0

**Pull request merged**